ui/src/components/Search.vue

```diff
@@ -175,23 +209,47 @@ export default {
         vfilter => vfilter.filter === filter
       );
       return validFilter.type === "boolean";
+    },
+    setFilter(item) {
```

Comment:
We need some unit tests to check that when any the possible filters are clicked, the input is correctly set.

Reply:
I added the unit tests on `search.spec.js`.